system76driver/daemon.py

```diff
@@ -653,6 +658,20 @@ def run_headphone_volume_adjust(model):
     except Exception:
         log.exception('Error calling _run_headphone_volume_adjust(%r):', model)
 
+def _run_line_volume_toggle(model):
+    if model not in NEEDS_LINE_VOLUME_TOGGLE:
+        log.info('Line volume toggle not needed for %r', model)
+        return
+    log.info('Line volume toggle for %r', model)
+    subprocess.call(["amixer", "-c", "Generic", "set", "Line", "mute"])
+    subprocess.call(["amixer", "-c", "Generic", "set", "Line", "unmute"])
+
+def run_line_volume_toggle(model):
```

Comment:
Unless there's some particularly arcane Python I'm missing, this function doesn't seem to be called.

Reply:
Oh duh